dali/dfu/dfurun.cpp

```diff
@@ -1124,17 +1124,9 @@ class CDFUengine: public CInterface, implements IDFUengine
         // if maxConnection is not passed as a user option, check for a default defined in the config/environment.
         if (!opttree->hasProp("@maxConnections"))
         {
-            int configDefault = -1;
-#ifdef _CONTAINERIZED
-            configDefault = getComponentConfigSP()->getPropInt("@maxConnections", -1);
```

Comment:
NB: This has been moved to the 'expert' section (values schema for dfu updated).
I don't think is a backward compatibility issue, because:
a) In bare-metal it was only an option at a global level, but it was not available as an equivalent global setting in containerized before this change.
b) This configuration setting was introduced for bare-metal, I am almost certain it has not been used in cloud.